### DISCUSSION TITLE
Convergence

### DISCUSSION THREAD

**Discussion by Shahin-rmz** [markmap/markmap#219] [Q&A]
Hello,
First of all thanks for bringing such a nice piece of code to world.
I am studying my Uni with Markmap and it saves me lots of time.
I have a question though, how can I converge and merge some branches into 1?
something like this
![image](https://github.com/markmap/markmap/assets/32805117/e81e3c58-f580-4764-afd9-edd5e39fea3b)
 
Thanks

**Comment by souovan** [comment]
I think it was answered here https://github.com/markmap/markmap/discussions/100#discussion-3834221

**Reply by Shahin-rmz** [reply]
Thanks :(
.
